service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/DiagnosticReportMapper.java

```diff
@@ -271,19 +277,20 @@ private void buildNarrativeStatementForParticipants(DiagnosticReport diagnosticR
             var humanNames = buildListOfHumanReadableNames(diagnosticReport.getPerformer());
             String performerNarrativeStatement = buildNarrativeStatementForDiagnosticReport(
                 diagnosticReport.getIssuedElement(), CommentType.AGGREGATE_COMMENT_SET.getCode(),
-                PREPENDED_TEXT_FOR_PARTICIPANTS + humanNames
+                PREPENDED_TEXT_FOR_PARTICIPANTS + humanNames,null
```

Comment:
Again, a space could would good.

Reply:
An overload of `buildNarrativeStatementForDiagnosticReport` without the `confidentialityCode` calling the other method with `NULL` might be a nice refactor.

Reply:
Agreed, will log and raise in separate PR 😄